nzedb/Binaries.php

```diff
@@ -637,14 +637,20 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 		// If there was an error, try to reconnect.
 		if ($this->_nntp->isError($headers)) {
 
+			if ($this->first == $this->last) {
```

Comment:
I don't know if that was the intent with that ternary or not, but it makes reading the code more complicated in general, which is why we have the coding standard. 
IF the intent is for micro-optimising, that has to be considered separately and might override normal coding standards. That's a pretty big IF though.